controlpanel/frontend/jinja2/webapp-detail.html

```diff
@@ -81,6 +89,7 @@ <h2 class="govuk-heading-m">
       <tbody class="govuk-table__body">
         {% for customer in customers %}
         <tr class="govuk-table__row">
+          <td class="govuk-table__cell">{{ row_selector('customer', customer.user_id) }}</td>
```

Comment:
this `row_selector` was already there right? I don't see in the diff but I can see it in the repo.

Reply:
yeah @andyhd already started on this stuff - I just finished it